ground-control/migrator/main.go

```
@@ -0,0 +1,109 @@
+	}
+
+	// Check if the output is "1" (indicating the database exists)
+	if string(output) == "1\n" {
+		return true, nil
+	}
+
+	return false, nil
+}
+
+// Create a new database if it doesn't exist
+func createDatabase(dbName, dbUser, dbPassword string) error {
```

golangci-lint (1.64.8)

func `createDatabase` is unused

(unused)